src/librustdoc/lib.rs

```diff
@@ -253,8 +257,19 @@ pub fn main_args(args: &[String]) -> isize {
     let markdown_input = input.ends_with(".md") || input.ends_with(".markdown");
 
     let output = matches.opt_str("o").map(|s| PathBuf::from(&s));
+    let css_file_extension = matches.opt_str("e").map(|s| PathBuf::from(&s));
     let cfgs = matches.opt_strs("cfg");
 
+    let nightly_options = NightlyOptions::get(&matches);
+
+    if let Some(ref p) = css_file_extension {
+        nightly_options.check_enabled("unstable-options", "extend-css");
```

Comment:
This specifically needs to not exist. The problem here is that this is an ad-hoc check to see if unstable options are enabled which is brittle and easy to forget. In the compiler you'll notice that the `optflag` *constructor* is tagged with a stability attribute and the same needs to happen in rustdoc.